add a list for ops to be added

cccclai opened the pull request:
Summary: As title, add a list of ops to be supported without erroring out.

Differential Revision: D61002938

haowhsu-quic reviewed (approved):
Thank you for adding this.

chiwwang reviewed (approved):
Looks good for me. Thanks!

guangy10 commented:
So with this fix, it will unblock the AOT path but we will defer the actual impl of that op later, correct?

cccclai commented:
> later

yeah that's right

cccclai committed:
add a list for ops to be added (#4624)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/4624

As title, add a list of ops to be supported without erroring out.

Reviewed By: guangy10, kirklandsign

Differential Revision: D61002938
